src/main/resources/assets/jei/lang/fr_fr.json

```diff
@@ -122,7 +122,8 @@
   "gui.jei.category.itemInformation": "Information",
 
   "_comment": "Messages",
-  "jei.message.ftbguilib": "Installez FTB GUI Library pour accéder aux configurations en jeu",
+  "jei.message.ftblibrary": "Install FTB Library to access ingame config",
+  "jei.message.ftbguilib": "Installez FTB Library pour accéder aux configurations en jeu",
```

Comment:
Are you sure this is what it should be? ;)

Reply:
oops, good catch thanks